lua/nvim_lsp/health.lua

```diff
@@ -0,0 +1,22 @@
+local M = {}
+function M.check_health()
+  local configs = require 'nvim_lsp/configs'
+
+  for _, top_level_config in pairs(configs) do
+    -- the folder needs to exist
+    local new_config = top_level_config.make_config(".")
```

Comment:
If you do not [move this process](https://github.com/neovim/nvim-lsp/pull/120/files#diff-bb1a5a30518e3cb9d6bb43af411ddbf5R151), it should work without any arguments.